deps/cpython/build_python.bat

```diff
@@ -48,7 +50,10 @@ echo "/p:TclVersion=%TCL_VERSION%" >> %response_file%
 :: won't be built.
 call %sourcedir%\PCbuild\build.bat -e --pgo
 
-if ERRORLEVEL 1 exit /b %ERRORLEVEL%
+if %errorlevel% neq 0 (
+   goto :cleanup
+   set script_errorlevel=%errorlevel%
```

Comment:
this captures the status code of the `cleanup` subroutine, is it what you want?

Reply:
Should be fixed now.